### PR TITLE
[inventory] Add documentation

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: common/v1/autogen.md [hunk 1]
```diff
@@ -53,16 +53,18 @@
...
 
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| asset_tag | [string](#string) |  |  |
-| serial_number | [string](#string) |  |  |
-| vendor | [string](#string) |  |  |
-| version | [string](#string) |  |  |
-| product | [string](#string) |  |  |
+| asset_tag | [string](#string) |  | Number of a null-terminated string |
```
Comment (seroyer) [inline common/v1/autogen.md:63]: Should these be "Number in a null-terminated string"?

File: common/v1/autogen.md [hunk 2]
```diff
@@ -72,13 +74,17 @@
 <a name="opi_api-inventory-v1-CPUInfo"></a>
 
 ### CPUInfo
-
+Processor Information (Type 4)
+ The information in this structure (see Table 21) defines the attributes of a single processor; a separate
+ structure instance is provided for each system processor socket/slot. For example, a system with an
+ IntelDX2™ processor would have a single structure instance while a system with an IntelSX2™ processor
+ would have a structure to describe the main CPU and a second structure to describe the 80487 co1054 processor.
```
Comment (seroyer) [inline common/v1/autogen.md:81]: I love this trip down memory lane

**Review by seroyer — APPROVED** [review]
Very helpful!

File: common/v1/autogen.md (hunk 1 above)
Comment (glimchb) [reply common/v1/autogen.md:63]: yeah, this is how https://www.dmtf.org/sites/default/files/standards/documents/DSP0134_3.6.0.pdf described them
we should iterate on this later on to deep dive and come up with better description

**Pull request merged**